docs/platforms/javascript/common/user-feedback/index.mdx

```diff
@@ -38,24 +37,31 @@ By default, this will insert the widget into the bottom right corner of your web
 On SDK version 8.0.0 and above, users can send screenshots with their feedback. If you're self-hosting, you also need release 24.4.2 and above. You can configure this using the `enableScreenshot` option, by default it is set to `true`. Screenshots aren't supported on mobile devices, so the screenshot button will be hidden automatically in this case.
 
 <Alert level="">
-  Screenshots use your [attachments quota](/pricing/quotas/manage-attachments-quota). All plans come with 1GB of attachments, which is approxiamately 2500 screenshots.
+  Screenshots use your [attachments
```

Comment:
was this one also automatically added?

Reply:
Great catch! I accidentally committed this file, it has nothing to do with this PR/issue 😅